www/data_service/src/md_demo/demo.controller.coffee

```diff
@@ -0,0 +1,4 @@
+class Demo extends Controller('app')
+    constructor: (dataService) ->
+        @builders = dataService.getBuilders()
```

Comment:
I don't see how this api can work with data binding.
Dataservice must maintain a list of watcher of the builders, so that it can stop listening for events on builders.